### ISSUE TITLE
About deep transfer learning experiment protocol

### ISSUE THREAD

**Issue by HongxuanLi-CB** [jindongwang/transferlearning#174]
Thank you for taking time out of your busy schedule to read this.
I'm an undergraduate,and new on deep transfer learning.I have three questions about the experiment in transfer learning:
1. The target data also join in training the model,should we divide the target data into training set and test set (in finetune_AlexNet_ResNet)?
![image](https://user-images.githubusercontent.com/63340444/79840210-34d13a80-83e8-11ea-92b5-eaf0866d30c9.png)
2. My knowledge is too shallow to understand the protocol in this screenshot.Why is the test set build like this:
![image](https://user-images.githubusercontent.com/63340444/79840808-04d66700-83e9-11ea-9c1f-d8fb2a6187d3.png)
3. I run the DDC code,and the accuracy is too high,looks like the accuracy in the train set.And the results of linear kernel and rbf kernel have a big gap(A->W:linear about 50%,rbf about 90%).
![image](https://user-images.githubusercontent.com/63340444/79841056-70b8cf80-83e9-11ea-9296-64f3af554f4c.png)
Thanks!!!!!!!!!!!!!

**Comment by HongxuanLi-CB** [comment]
I am sorry for my mistake,there is something wrong with the 3rd question,I forget to change the hyperparameter "num_class",the accuracy is not correct.

**Comment by jindongwang** [comment]
@HongxuanLi-CB 1. Yes。 2. It is built by following existing work. Just call them some common CV tricks to improve the performance.

**Comment by HongxuanLi-CB** [comment]
![2B207B48-33C9-4C3C-8D96-5CE05029D6D5](https://user-images.githubusercontent.com/63340444/80053662-066e6f00-8550-11ea-8080-295be02ce4b0.jpeg)
I use this transform for both source and target data，and then I split the transformed target data into train and test set，the transformed source data into train and validate set. Is this operation correct for an experiment ？
Thanks！！！！！

**Comment by jindongwang** [comment]
@HongxuanLi-CB  Yes. Actually you can follow the settings of our previous code such as DDC and DAAN, etc.

**Comment by HongxuanLi-CB** [comment]
Thank you for your work in transfer learning!
Without your code ,data and blogs,it's going to be such a rough ride for us beginners!!!!!!
